http/headers/cross-origin-resource-policy.json

```
@@ -9,14 +9,14 @@
               "version_added": "73",
               "notes": [
                 "Until version 75, downloads for files with this header would fail in Chrome. See <a href='https://crbug.com/952834'>bug 952834</a>.",
-                "From version 80, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>."
+                "From version 80, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>. Partial loading was disabled in v86, mitigating this bug."
```

Can adjust this a bit to put the applicable versions up front?

```suggestion
                "From version 80 to 85, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>. From version 86, partial PDF loading is disabled."
```